examples/node/lib/rest/Twilio.ts

```diff
@@ -12,14 +17,14 @@ class Twilio {
     return new FlexApiBase(this);
   }
 
-  request(opts: RequestOpts) {
-    return new RequestClient().request({
-      method: opts.method,
-      uri: opts.uri,
-      params: opts.params,
-      data: opts.data,
-    });
-  }
+  // request(opts: RequestOpts) {
```

Comment:
Should uncomment/roll this back.

Reply:
I tired using this initially but kept getting a "Invalid security scheme used" issue. Even when I added in the username and password options to the request

Reply:
Either way, should drop the commented-out code.